subsys/net/ip/net_private.h

```diff
@@ -34,6 +34,11 @@ union net_mgmt_events {
 #if defined(CONFIG_NET_IPV6) && defined(CONFIG_NET_IPV6_MLD)
 	struct net_event_ipv6_route ipv6_route;
 #endif /* CONFIG_NET_IPV6 && CONFIG_NET_IPV6_MLD */
+#if defined(CONFIG_NET_L2_WIFI_MGMT)
+#if defined(CONFIG_WIFI_MGMT_RAW_SCAN_RESULTS)
+	struct wifi_raw_scan_result raw_scan_result;
+#endif /* CONFIG_WIFI_MGMT_RAW_SCAN_RESULTS */
```

Comment:
I meant, there is already `#if defined(CONFIG_NET_L2_WIFI_MGMT)` above, so just move it there.